### PR TITLE
[Part #1]: Ensures the modules documentation follows the correct formats

### PR DESCRIPTION
**Pull request by alinabuzachis**
##### SUMMARY

Ensures the modules documentation follows the correct formats https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#semantic-markup-within-module-documentation

More PRs are yet to come

 
##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by tremble — APPROVED** [review]
Some minor additional suggestions, but the change looks good.

File: plugins/modules/ec2_ami.py [hunk 1]
```diff
@@ -91,7 +91,7 @@
           - Suppresses the specified device included in the block device mapping of the AMI.
       volume_type:
         type: str
-        description: The volume type.  Defaults to C(gp2) when not set.
+        description: The volume type. Defaults to v(gp2) when not set.
```
Comment (tremble) [inline plugins/modules/ec2_ami.py:94]: Does this need to be V() ?

**Review by tremble — APPROVED** [review]

File: plugins/modules/backup_plan.py [hunk 2]
```diff
@@ -83,7 +83,7 @@
           delete_after_days:
             description: Specifies the number of days after creation that a recovery
               point is deleted. Must be greater than 90 days plus
```
Comment (abikouo) [inline plugins/modules/backup_plan.py:85]:
```suggestion
              point is deleted. Must be greater than V(90) days plus
```
?

File: plugins/modules/backup_plan.py [hunk 3]
```diff
@@ -115,14 +115,14 @@
               delete_after_days:
                 description: Specifies the number of days after creation that a
                   recovery point is deleted. Must be greater than 90 days plus
```
Comment (abikouo) [inline plugins/modules/backup_plan.py:117]:
```suggestion
                  recovery point is deleted. Must be greater than V(90) days plus
```

**Review by abikouo — APPROVED** [review]

File: plugins/modules/ec2_ami.py (hunk 1 above)
Comment (tremble) [reply plugins/modules/ec2_ami.py:94]:
```suggestion
        description: The volume type. Defaults to V(gp2) when not set.
```

**Review by tremble — APPROVED** [review]

**Pull request merged**